phpseclib/Crypt/RSA.php

```diff
@@ -2196,18 +2258,20 @@ function decrypt($ciphertext)
      * @see self::verify()
      * @access public
      * @param string $message
+     * @param int $padding
```

Comment:
Should this be `@param int $padding optional`, as seems to be the convention for optional parameters?